test/functional/test-url-replacements.js

```diff
@@ -19,17 +19,30 @@ import {createIframePromise} from '../../testing/iframe';
 import {urlReplacementsFor} from '../../src/url-replacements';
 import {markElementScheduledForTesting} from '../../src/custom-element';
 import {installCidService} from '../../src/service/cid-impl';
+import {installViewerService} from '../../src/service/viewer-impl';
 import {setCookie} from '../../src/cookies';
-
+import * as sinon from 'sinon';
```

Comment:
There was some weird problem with it and fake timers. Not sure if that was addressed on sinon side. @cramforce do you remember?